.travis.yml

```diff
@@ -13,6 +13,7 @@ install:
   - make docker.build
 
 script:
+  - docker-compose -f docker-compose.yml -f docker-compose.travis.yml run app "make migrate"
```

Comment:
Previously build failed, I believe because we weren't applying the migrations. This line added to migrate before other steps. But not we are getting a separate travis build error: https://travis-ci.org/openoakland/woeip/builds/468789778#L991

Seems like the ``app`` container is trying to connect to the ``db`` container before the db is finished initializing. . This is happening even with the docker-compose setting ``app: { depends_on: [ db ]}``. I think the problem is ``depends_on`` only checks that the container has started -- there are additional setup steps that the ``db`` container needs to run before ``app`` can contact it without crashing.

Reply:
https://docs.docker.com/compose/startup-order/

Reply:
You will probably want to modify the entry point script to (a) wait for Postgres to be ready and (b) support passing in the script commands. I can take this on sometime tomorrow.

Reply:
...."This can go away" sounds like a mobster threat